### ISSUE TITLE
[data-model] Implement multiple ordering in the data model definition

### ISSUE THREAD

**Issue by thekangaroofactory** [thekangaroofactory/kitems#239]
Basic date ordering has been implemented with #224 
However, a stronger ordering feature could be implemented through the data model definition.

Add a logical sort attribute to the data model, with true for column that should be used to order the items.
Add a logical decrease attribute to the data model, with true if the sort should be done in decreasing order.

Check if it's possible to have multiple sort columns, then replace logical by integer or add an integer sort_priority attribute.
dd[order(-dd[,4], dd[,1]), ]
dd[with(dd, order(-z, b)), ]

**Comment by thekangaroofactory** [comment]
Be careful that there is an ordering stuff in filtered_items !!!

```
    # -- Filtered item table view
    filtered_items <- reactive({

      # -- check
      if(!is.null(filter_date())){

        cat(MODULE, "Updating filtered item view \n")

        # -- Get items
        items <- k_items()

        # -- Apply date filter
        items <- items[items$date >= filter_date()[1] & items$date <= filter_date()[2], ]

        # -- Apply ordering (WTF !??)
        items <- items[order(items$date, decreasing = TRUE), ]
```

**Comment by thekangaroofactory** [comment]
The mechanism has been implemented inside data model & server code
Input in the admin UI to setup sort info will be implemented by #281
